### PR TITLE
327 mentee form to iFrame exploration

### LINKED ISSUE #327: Explore how we can port mentee form to an iFrame

### PR DESCRIPTION
**Pull request by katmartel**
For the sake of readability, it might be easier to view my file directly rather than reading it from the 'Files Changed' tab.

### CODE REVIEW

**Review by aleksssk — APPROVED** [review]
Great work!

**Review by wrc9817 — APPROVED** [review]
That's a lot of research. good job.

File: documentation/exploration/menteeForm-to-iFrame/jan20-27-sprint.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        <iframe
+            src="../MenteeForm/MenteeForm.js" //changing 'src="MenteeForm"' produces the same result
+            height="500"
+            width="1100"
+            title="testIframe"
+          />
+```
+
+- As an aside: Yes, if you navigate to the 'MentorTutorial' component within the embedded iFrame, what I would deem as 'Menteeform-ception' occurs (or, I guess since the suffix '-ception' doesn't technically mean 'something nested within itself', I should really just say that recursion is the result).
+  ![](menteeform-ception.png)
```
Comment (jpabadir) [inline documentation/exploration/menteeForm-to-iFrame/jan20-27-sprint.md:37]: Good terminology.

**Review by jpabadir — APPROVED** [review]
Thanks for this research. I would also need to research this to suggest my own solutions.

**Pull request merged**